src/main/kotlin/com/cognifide/gradle/aem/common/file/transfer/FileTransferManager.kt

```diff
@@ -76,7 +75,14 @@ class FileTransferManager(private val aem: AemExtension) : FileTransfer {
         }
 
         GFileUtils.mkdirs(target.parentFile)
-        FileOperations.lockOperation(target) { handling(dirUrl).downloadFrom(dirUrl, fileName, it) }
+
+        val tmp = File(target.parentFile, "${target.name}$TMP_SUFFIX")
```

Comment:
no *.lock files, but downloading to *.tmp files then renaming without such suffix (in a same way as it is Chrome doing)